packages/dapp/src/components/Navbar/Menu/WalletMenu/WalletMenu.tsx

```diff
@@ -17,6 +16,7 @@ import { SpotButton } from '@transferto/shared/src/atoms';
 import { openInNewTab, walletDigest } from '@transferto/shared/src/utils';
 import { useCallback, useEffect, useMemo, useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import { useChains } from '../../../../hooks/useChains';
```

Comment:
Also, you know there is something wrong when you need to make `../../../../` imports 😄